OptimizelySDK/Bucketing/DecisionService.cs

```diff
@@ -662,5 +663,31 @@ private Result<string> GetBucketingId(string userId, UserAttributes filteredAttr
 
             return Result<string>.NewResult(bucketingId, reasons);
         }
+
+        /// <summary>
+        /// Finds a validated forced decision.
+        /// </summary>
+        /// <param name="context">Object containing flag and rule key of which forced decision is set.</param>
+        /// <param name="flagDecision">Object containing forced decision based on the provided context.</param>
+        /// <param name="config">The Project config.</param>
+        /// <returns>A result with the variation</returns>
+        public Result<Variation> ValidatedForcedDecision(OptimizelyDecisionContext context, OptimizelyForcedDecision forcedDecision, ProjectConfig config, string userId)
```

Comment:
What about passing userContext. This can be simpler and extensible with usercontext?
```suggestion
        public Result<Variation> ValidatedForcedDecision(OptimizelyDecisionContext context, OptimizelyUserContext user, ProjectConfig config)
```